cryptol-remote-api/cryptol-remote-api.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.4
 name:                cryptol-remote-api
-version:             3.0.0.99
+version:             3.0.1.99
```

Comment:
A comment about process: the convention I've been using for releases (in this case, a PyPI release) is to make a commit that bumps the version numbers to X, and then a subsequent commit which bumps the commit to X.99, thereby turning it into a development version for a future X+1 release. That way, we can tag the commit that bumps it to X.

Do you mind doing that here? That is, have one commit that bumps this to 3.0.1 (which we tag with something like `git tag cryptol-remote-api-3.0.1`), plus an subsequent commit on top of that which bumps it to 3.0.1.99 (the development version for a future 3.0.2/3.1.0 release, depending on how the 3.* release series evolves).

Reply:
Ah, that's a good system! I added commit ae4cb538c9dd1e94438dc8602076bb1c958f3c5d, which bumps the versions to `3.0.1.00`, commit a0903546187826e16aec2ec797da5db996f8e6af, which bumps the versions to `3.0.1.99`, and the tag [cryptol-remote-api-3.0.1](https://github.com/GaloisInc/cryptol/releases/tag/cryptol-remote-api-3.0.1) which is set to the first commit. Does that look right?